Documentation/gettingstarted/encryption.rst

```diff
@@ -12,9 +12,9 @@ Transparent Encryption
 
 This guide explains how to configure Cilium to use IPsec based transparent
 encryption using Kubernetes secrets to distribute the IPsec keys. After this
-configuration is complete all traffic between Cilium-managed endpoints, as well
-as Cilium managed host traffic, will be encrypted using IPsec. This guide uses
-Kubernetes secrets to distribute keys. Alternatively, keys may be manually
+configuration is complete all traffic between Cilium-managed endpoints,
+as well as Cilium managed host traffic, will be encrypted using IPsec.
```

Comment:
I think you'll need to repush for the Signed-off-by anyway, so might be worth fixing this small typo (not introduced by your PR):
```suggestion
as well as Cilium-managed host traffic, will be encrypted using IPsec.
```